core/src/main/scala/com/salesforce/op/filters/PreparedFeatures.scala

```diff
@@ -169,7 +169,8 @@ private[filters] object PreparedFeatures {
       case SomeValue(v: DenseVector) => Map((name, None) -> Right(v.toArray.toSeq))
       case SomeValue(v: SparseVector) => Map((name, None) -> Right(v.indices.map(_.toDouble).toSeq))
       case ft@SomeValue(_) => ft match {
-        case v: Text => Map((name, None) -> Left(v.value.toSeq.flatMap(tokenize)))
+        // case v: Text => Map((name, None) -> Left(v.value.toSeq.flatMap(tokenize)))
```

Comment:
We are no longer tokenzing text during data prep?

Reply:
Whoops, that was for testing - forgot to take it out.